pkg/kubeapiserver/authorizer/config.go

```diff
@@ -91,7 +95,15 @@ func (config Config) New() (authorizer.Authorizer, authorizer.RuleResolver, erro
 				config.VersionedInformerFactory.Core().V1().PersistentVolumes(),
 				config.VersionedInformerFactory.Storage().V1().VolumeAttachments(),
 			)
-			nodeAuthorizer := node.NewAuthorizer(graph, nodeidentifier.NewDefaultNodeIdentifier(), bootstrappolicy.NodeRules())
+			nodeAuthorizer := withNotSyncedReason{
```

Comment:
don't wrap the node authorizer for this, the only time it returns empty reasons is for non-nodes and when it was a type the node authorizer didn't have an opinion about